unit1.pas

```diff
@@ -482,6 +484,63 @@
   form11.ShowModal;
 End;
 
+procedure TForm1.MenuItem48Click(Sender: TObject);
+Var
+  i, j, LvlIdx, LvlCnt, l, cnt, xCnt, yCnt, h, w: integer;
+  f: Double;
+  LvlGrph: TLvlGraph;
+  LvlNode: TLvlGraphNode;
+  node: TNode;
+begin
+  // Order All Nodes with LvlGraph
+  cnt := GraphBox1.Graph.NodeCount;
+  xCnt := GraphBox1.Width Div (DefaultDistanceX);
+  yCnt := GraphBox1.Height Div (DefaultDistanceY);
+  f := cnt / (xCnt * yCnt) * 1.5;
+  xCnt := round(xCnt * f);
+
+  LvlGrph := TLvlGraph.create;
+  For i := 0 To GraphBox1.Graph.NodeCount - 1 Do
+    LvlGrph.GetNode(GraphBox1.Graph.Node[i].Caption, True);
+  For i := 0 To GraphBox1.Graph.NodeCount - 1 Do
+    For j := 0 To high(GraphBox1.Graph.Node[i].Edges) Do
+      LvlGrph.GetEdge(
+        GraphBox1.Graph.Node[GraphBox1.Graph.Node[i].Edges[j].StartIndex].Caption,
+        GraphBox1.Graph.Node[GraphBox1.Graph.Node[i].Edges[j].EndIndex].Caption,
+        True
+      );

```

Comment:
For j := 0 To high(GraphBox1.Graph.Node[i].Edges) Do Begin
      If GraphBox1.Graph.Node[GraphBox1.Graph.Node[i].Edges[j].StartIndex].Visible And
        GraphBox1.Graph.Node[GraphBox1.Graph.Node[i].Edges[j].EndIndex].Visible Then Begin
        LvlGrph.GetEdge(
          GraphBox1.Graph.Node[GraphBox1.Graph.Node[i].Edges[j].StartIndex].Caption,
          GraphBox1.Graph.Node[GraphBox1.Graph.Node[i].Edges[j].EndIndex].Caption,
          True
          );
      End;